templates/sfn/ingest_sfn_definition.json.tpl

```diff
@@ -462,13 +462,24 @@
               {
                 "Variable": "$.wasReconciled",
                 "BooleanEquals": true,
-                "Next": "Do nothing"
+                "Next": "Publish reconciliation update to dr2-notifications"
               }
             ],
             "Default": "Throw Reconciler job error"
           },
-          "Do nothing": {
-            "Type": "Pass",
+          "Publish reconciliation update to dr2-notifications": {
+            "Type": "Task",
+            "Resource": "arn:aws:states:::sns:publish",
+            "Parameters": {
+              "Message": {
+                "reconciliationUpdate": "Asset was reconciled",
+                "assetId.$": "$.AssetInfo.assetId",
+                "messageId": "$.messageId",
+                "parentMessageId": "$.parentMessageId",
+                "executionId": "$.executionId"
```

Comment:
Can you pop these into a nested `properties` object? We've got more to do on the schema but this is a good start.

Reply:
Also missing the trailing `.$` on the keys.